### ISSUE TITLE
Huawei P8 lite: Something went wrong: 10: Unable to validate key file signature: Pipe is closed

### ISSUE THREAD

**Issue by Petk12** [corona-warn-app/cwa-app-android#747]
When I start the Corona App, it shows a normal screen with a grey message for a glimpse, then shows the stack trace instead of the usual green or other message. It is reproducible. Version 1.0.4.

![IMG_2387](https://user-images.githubusercontent.com/67408229/85668705-56d48b00-b6bf-11ea-9dcc-7d4e4fa4d7ca.JPG)
![IMG_2386](https://user-images.githubusercontent.com/67408229/85668718-59cf7b80-b6bf-11ea-9c3a-007253ab0548.JPG)

## Technical details

- Mobile device: Huawei P8 lite 2GB main memory 1,4GB free, 3,16GB free flash memory [ALE-L21]
- Android version: 6.0
- kernel version 3.10.86-g33ff982android@localhost #2 Tue Mar 6 17:39:54 CST 2018

**Comment by BranKohlschreiber** [comment]
i got the same bug, 
ALE-L21 
App Version 1.0.4

**Comment by jakobmoellersap** [comment]
Hi, thanks for reporting the issue. We are investigating the issue and will come back once we have more results.

**Comment by jakobmoellersap** [comment]
Duplicate of #737

**Comment by styxxx** [comment]
Since you closed #737 for comments I'm posting here, since this might help in some way:
I'm affected too but only after upgrading CWA to the latest version. Didn't happen before so this might not be a coincidence.  
Backview BV6000, Android 6
CWA 1.0.4, Versionscode: 31
